diff --git a/src/d_main.cpp b/src/d_main.cpp
index e6f5e9081..67b1ffd20 100644
--- a/src/d_main.cpp
+++ b/src/d_main.cpp
@@ -3939,8 +3939,11 @@ void SignalHandler(int signal)
 int GameMain()
 {
 	// On Windows, prefer the native win32 backend.
+	// On macOS, prefer the native Cocoa backend.
 	// On other platforms, use SDL until the other backends are more mature.
 	auto zwidget = DisplayBackend::TryCreateWin32();
+	if (!zwidget)
+		zwidget = DisplayBackend::TryCreateCocoa();
 	if (!zwidget)
 		zwidget = DisplayBackend::TryCreateSDL2();
 	if (!zwidget)
